frontend/src/element/signup.ts

```diff
@@ -29,6 +29,22 @@ import {
 export class Signup extends LitElement {
   private readonly stripe = loadStripe(window.STRIPE_KEY);
 
+  @query('[name="preferred-name"]')
+  preferredName!: HTMLInputElement;
+  @query('[name="preferred-language"]')
+  preferredLanguage!: HTMLInputElement;
+  @query('[name="personal-email"]')
+  personalEmail!: HTMLInputElement;
+  @query('[name="employement-type"]')
```

Comment:
typo, here and in other repetitions of this class or property name:

```suggestion
  @query('[name="employment-type"]')
```